Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/where_are_artifacts_downloaded_and_how_can_i_control_that.md

---
title: Where are artifacts downloaded, and how can I control that?
toc_hide: true
type: docs
tags:
  - artifacts
  - environment variables
---

> By default, artifacts download to the `artifacts/` folder. You can change this by:
> 
> ```python
> wandb.Artifact().download(root="<path_to_download>")
> ```
> 
> Or by setting an environment variable:
> 
> ```python
> import os
> os.environ["WANDB_ARTIFACT_DIR"] = "<path_to_download>"
> ```

Reviewer: ```suggestion
By default, artifacts download to the `artifacts/` folder. To change the location:

- Pass it to [`wandb.Artifact().download`]({{< relref "/ref/python/public-api/api.md" >}}):

    ```python
    wandb.Artifact().download(root="<path_to_download>")
    ```

- Set the `WANDB_ARTIFACT_DIR` [environment variable]({{< relref "/guides/track/environment-variables.md" >}}):

    ```python
    import os
    os.environ["WANDB_ARTIFACT_DIR"] = "<path_to_download>"
    ```
```
